awxkit/awxkit/api/pages/credential_input_sources.py

```diff
@@ -4,6 +4,7 @@
 
 
 class CredentialInputSource(base.Base):
+    NATURAL_KEY = ('metadata', 'input_field_name')
```

Comment:
The credential input source does not need a natural key when you are only importing and exporting using `DEPENDENT_EXPORT`.

Reply:
Interesting, 
Before I added the `NATURAL_KEY` I encountered an error of:
```
This object does not have a natural key: credentials_input_sources
```
And the input_sources field inside the exported json was empty.
Could it be I am missing something?

Reply:
After testing again, It seems the `NATURAL_KEY` are indeed not needed. Thank you!

Reply:
@jbradberry I think we should add the `NATURAL_KEY` to the CredentialInputSource because the api uses it to reach `/api/v2/credential_input_sources` (unlike `SurvySpev`, which the api uses to reach a relative survey).

Though I do suggest to change `NATURAL_KEY = ('metadata', 'input_field_name')` to simply `NATURAL_KEY = ('input_field_name')`

Reply:
`('input_field_name')` is not a tuple.  If it is going to be there, it needs to be a tuple.

Reply:
Also, `DEPENDENT_EXPORT` and needing a natural key are mutually exclusive.

Reply:
never mind, more detail coming shortly

Reply:
@jbradberry, funilly enough, I resolved this issue by changing from DEPENDENT_EXPORT to DEPENDENT_NONEXPORT. Now there is no need for the NATURAL_KEY.
I think the DEPENDENT_EXPORT relies on the existance of a NATURAL_KEY, but I may be wrong.

I have tested export and import in the new commits, would love it if you take a look.